test/locationscale.jl

```diff
@@ -173,4 +173,13 @@ end
         test_location_scale_discretenonparametric(_rng, -1//4, 1//3, (-10):(-1), probs)
         test_location_scale_discretenonparametric(_rng, 6//5, 3//2, 15:24, probs)
     end
+
+    @test_nowarn ad_norm = AffineDistribution(1.0, 1, Normal())
+    @test ad_norm isa AffineDistribution{Float64, Continuous, Normal{Float64}}
+
+    @test_warn ls_norm = LocationScale(1.0, 1, Normal())
+    @test ls_norm isa LocationScale{Float64, Continuous, Normal{Float64}}
+    @test ls_norm isa AffineDistribution{Float64, Continuous, Normal{Float64}}
```

Comment:
These tests fail, `@test_warn` doesn't exist 🙃 I suggest using my suggestion above:
```suggestion
    @test (@test_deprecated LocationScale(1.0, 1, Normal())) isa LocationScale{Float64, Continuous, Normal{Float64}}
```

Reply:
Ah, test_warn came later than Julia 1.0

Reply:
Ah no, I skimmed through the logs too quickly: It exists but it requires an error message to match: https://docs.julialang.org/en/v1/stdlib/Test/#Test.@test_warn (BTW it doesn't cover logs generated with the logging system but only stderr - does it actually cover deprecation warnings?)

In any case, it is simpler and more direct to use `@test_deprecated`.